bin/cron/sync_contact_rollups

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env ruby
+
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require src_dir 'database'
+require_relative('../../dashboard/config/environment')
+require 'cdo/properties'
+
+# Connection to read from reporting database.
+PEGASUS_REPORTING_DB_READER = sequel_connect(CDO.pegasus_reporting_db_reader, CDO.pegasus_reporting_db_reader)
+
+# Connection to read from production database.
+PEGASUS_DB_READER = sequel_connect(CDO.pegasus_db_writer, CDO.pegasus_db_reader)
+
+# Connection to write to production database. (We use a 2nd production database connection because we need to do writes
+# while continuing to iterate through results of an earlier read query from that same database.)
+PEGASUS_DB_WRITER = sequel_connect(CDO.pegasus_db_writer, CDO.pegasus_db_reader)
+
+# Columns to disregard
+EXCLUDED_COLUMNS = %w(id pardot_id pardot_sync_at updated_at).freeze
```

Comment:
Per our slack discussion, am freezing `EXCLUDED_COLUMNS` already. For DB constants I am following the convention of the rest of the code.